intranet/templates/polls/results.html

```diff
@@ -68,8 +80,8 @@
             &nbsp; <a href="{% url 'poll_csv_results' poll.id %}" class="button small-button print-hide">Download as CSV</a>
         {% endif %}<br>
         <h2>Results: {{ poll }}</h2>
-        <p>&nbsp; {{ poll.get_voted_string }}</p>
-
+        <p>&nbsp; {{ poll.get_voted_string }}</p><br>
+        <p>&nbsp; Show results by gender?<input type="checkbox"></p><br>
```

Comment:
You probably should give this an ID and then reference it in the JS instead of filtering on `[type="checkbox"]` in case we want to add another filter in the future.

Reply:
Fixed